packages/core/src/sass/account-wizard.scss

```diff
@@ -189,30 +189,18 @@
                 margin-top: 1.6rem;
             }
         }
-        // fix for safari bug with autoheight wrapper not working properly for calculating modal footer offset
-        @media not all and (min-resolution: 0.001dpcm) {
-            // stylelint-disable-line
-            @supports (-webkit-appearance: none) and (stroke-color: transparent) {
-                @include desktop {
-                    max-height: calc(100% - 48px);
-                }
-                @include tablet {
-                    max-height: calc(100% - 48px);
-                }
-            }
-        }
         @include desktop {
-            height: 100%;
             display: grid;
```

Comment:
if desktop and tablet are the same, we can put this in the original class at line 154 instead

Reply:
But we don’t want this styles on mobile. So we should use "@include tablet-up".

Reply:
thats right, you can use tablet-up and remove desktop and mobile altogether